docs/Enable S3 support in SSM.md

```diff
@@ -0,0 +1,102 @@
+# Enable S3 support in SSM
+
+SSM has already supported S3 in copy action. Now we can set a remote S3 path as a destination in 'copy2s3' action.
+
+## How to enable S3 support in HDFS（optional）
+
+This is an optional step. To test S3 support of SSM, we can use the command of HDFS to check the file in remote S3 cluster(AWS S3). The hadoop version is 2.7.
+
+### Solve the dependency
+
+It is easy enable S3 support of HDFS. First we need to copy some jar packages about `aws` and `jackson` from `${HADOOP_HOME}/share/hadoop/tools/lib}` to `${HADOOP_HOME}/share/hadoop/tools/common/lib`. 
+
+The jar dependencies we need to copy:
+
+```
+aws-java-sdk-1.7.4.jar
+hadoop-aws-2.7.4.jar
+jackson-annotations-2.2.3.jar
+jackson-core-2.2.3.jar
+jackson-core-asl-1.9.13.jar
+jackson-databind-2.2.3.jar
+jackson-jaxrs-1.9.13.jar
+jackson-mapper-asl-1.9.13.jar
+jackson-xc-1.9.13.jar
+```
+
+We can find these jar dependencies in `${HADOOP_HOME}/share/hadoop/tools/lib}` by using commands like this:
+
+```shell
+ll | grep jackson
+ll | grep aws
+```
+
+### Add configuration of AWS S3
+
+Then we add some configurations in `${HADOOP_HOME}/etc/core-site.xml`:
+
+```xml
+<property>
+        <name>fs.s3a.access.key</name>
+        <value></value>
+</property>
+<property>
+        <name>fs.s3a.secret.key</name>
+        <value></value>
+</property>
+```
+
+### Try to use HDFS command
+
+We can use the command `hdfs dfs -ls s3a://{YOU_PATH}/` to list the file in remote AWS S3 cluster.
+
+An example:
+
+```shell
+$ hdfs dfs -ls s3a://xxxctest/
```

Comment:
Replace `s3a://xxctest/` with {test_dir}, where {parameter name} is widely used for parameter.